pkg/capture/capture.go

```diff
@@ -160,6 +160,13 @@ func (c *Capture) run() (err error) {
 		return fmt.Errorf("failed to initialize capture: %w", err)
 	}
 
+	c.capLock = concurrency.NewThreePointLock(
+		concurrency.WithMemPool(memPool),
+		concurrency.WithTimeout(30*time.Second),
```

Comment:
You knew I was going to say it: stuff it into a const, or better, a config variable.